kairon/api/app/routers/bot/bot.py

```
@@ -1658,3 +1660,60 @@
+    update the mail config
+    """
+    request_dict = request_data.dict()
+    MailClassificationConfig.update_doc(**request_dict, bot=current_user.get_bot())
+    return {"message": "Config updated!"}
+
+
+
+@router.delete("/mail/config/{intent}", response_model=Response)
+async def del_soft_mail_config(
+        intent: str,
+        current_user: User = Security(Authentication.get_current_user_and_bot, scopes=DESIGNER_ACCESS)
```

Ruff

Do not perform function call `Security` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)